dsgrid/dataset/dataset_schema_handler_standard.py

```diff
@@ -40,10 +54,121 @@ def get_unique_dimension_rows(self):
         """Get distinct combinations of remapped dimensions, including id.
         Check each col in combination for null value."""
         dim_table = self._remap_dimension_columns(self._load_data_lookup).distinct()
-        self._check_null_value_in_unique_dimension_rows(dim_table)
-
+        check_null_value_in_unique_dimension_rows(dim_table)
         return dim_table
 
+    def make_project_dataframe(self):
+        # TODO: Can we remove NULLs at registration time?
```

Comment:
Not sure if we can completely remove null values give pivoted dim structure. Imagine a dataset where a county does not have a particular end use. We can always replace remaining null values with 0s. But it might affect mean calculation.

Reply:
Worth removing where id is null though.

Reply:
In the standard format for input datasets:
- load_data cannot have nulls
- load_data_lookup cannot have nulls for dimension columns (right?). If the data ID column has a null, that row has been intentionally excluded.

Reply:
right, the question is whether we should drop the id=null rows from load_data_lookup even though we painstakingly put them there previously

Reply:
Result from discussion:
- Strip the null rows from load_data_lookup at registration time and insert them into a new file. All future queries will never have to worry about null rows.
- Add a new feature where the dataset contributor can create that secondary file up front and register the pair.

Reply:
Resolving because of #148